### DISCUSSION TITLE
Notification distribution providers

### DISCUSSION THREAD

**Discussion by Xtrems876** [jocmp/capyreader#257] [Q&A] [answered]
Hello. Many apps explicitly state that their f-droid versions do not support notification distribution via google play services, and that a third-party distributor is then needed. Examples include many clients for mastodon.
Their counterparts on google play store however do support that and do not require a third party distributor.
I struggled to find documentation on this for this reader - and I use RSS readers explicitly for the benefit of being notified about various updates to things which are important to me but ones I cannot be bother to look up every day.
What is the case for this app? Will notifications reliably come if I install it through google play store?

**Comment by jocmp** [comment, marked as answer]
This app doesn't support notifications, so it's the same feature set for Google Play and F-Droid.

**Reply by sandboiii** [reply]
Are you not going to implement notifications? This misses the whole point of the rss feed - to notify about new things

**Reply by jocmp** [reply]
They're planned but not started. See #264
